src/client/app/shared/canvas/canvas.service.ts

```diff
@@ -0,0 +1,91 @@
+import { Injectable } from '@angular/core';
+import '../../shared/card/card';
+
+// TODO(eyuelt): where should this be defined?
```

Comment:
If "Point" is only used in the canvas service, then here is fine.

If not, probably closer to the data model layer.